DotNet.DocsTools/GitHubObjects/QuestIssue.cs

```diff
@@ -0,0 +1,385 @@
+            return ((actor is not null) ? actor.Login : "Ghost", 
+                ResponseExtractors.StringProperty(item, "bodyHTML"));
+            }
+        ).ToArray();
+
+        var points = ResponseExtractors.GetChildArrayElements(issueNode, "projectItems", item =>
+            StoryPointSize.OptionalFromJsonElement(item));
+        this.ProjectStoryPoints = [ ..points.Where(sz => sz is not null).ToArray()];
+
+        // check state. If re-opened, don't reference the (not correct) closing PR
+        this.ClosingPRUrl = ResponseExtractors.GetChildArrayElements(issueNode, "timelineItems", item =>
+            (item.TryGetProperty("closer", out var closer) && closer.ValueKind == JsonValueKind.Object) ?
+            ResponseExtractors.StringProperty(closer, "url")
+            : default).LastOrDefault(url => url is not null);
+
+        this.LinkText = $"""
+        <a href = "https://github.com/{organization}/{repository}/issues/{Number}">
+            {organization}/{repository}#{Number}
+        </a>
+        """;
+    }
+
+    /// <summary>
+    /// True if the issue is open.
+    /// </summary>
+    public bool IsOpen { get; }
+
+    /// <summary>
+    /// The issue author.
+    /// </summary>
+    public string FormattedAuthorLoginName { get; }
+
+    /// <summary>
+    /// The body of the issue, formatted as HTML
+    /// </summary>
+    public string? BodyHtml { get; }
+
+    /// <summary>
+    /// The list of assignees. Empty if unassigned.
+    /// </summary>
+    public Actor[] Assignees { get; }
+
+    /// <summary>
+    /// The list of labels. Empty if unassigned.
+    /// </summary>
+    public GitHubLabel[] Labels { get; }
+
+    /// <summary>
+    /// The list of comments. Empty if there are no comments.
+    /// </summary>
+    /// <remarks>
+    /// The tuple includes the author and the body, formatted as HTML.
+    /// </remarks>
+    public (string author, string bodyHTML)[] Comments { get; }
+
+    /// <summary>
+    /// The link text to this GH Issue.
+    /// </summary>
+    /// <remarks>
+    /// the link text is formatted HTML for the link to the issue.
+    /// </remarks>
+    public string LinkText { get; }
+
+    public DateTime UpdatedAt { get; }
+
+    /// <summary>
+    /// Pairs of Project name, story point size values
+    /// </summary>
+    public IEnumerable<StoryPointSize> ProjectStoryPoints { get; }
+
+    /// <summary>
+    /// The Closing PR (if the issue is closed)
+    /// </summary>
+    public string? ClosingPRUrl { get; }
+
+    /// <summary>
+    /// Retrieve the assigned name, if an MS employee
+    /// </summary>
+    /// <param name="ospoClient">The Open Source program office client service.</param>
+    /// <returns>The email address of the assignee. Null if unassigned, or the assignee is not a 
+    /// Microsoft FTE.</returns>
+    public async Task<string?> AssignedMicrosoftEmailAddress(OspoClient ospoClient)
```

Comment:
I'd recommend changing the name of this method to indicate that it's doing network I/O. Perhaps add a verb, like `Get` and a suffix like `Async`.